### PR TITLE
[dev] AB#86692 -- Ability to Upload Large Video Files

### PR DESCRIPTION
**Pull request by IamNotHuman**
added file type to inputs

### CODE REVIEW

**Review by groberts314 — APPROVED** [review]

**Review by groberts314 — APPROVED** [review]

**Review by raoufr — APPROVED** [review]

**Comment by raoufr** [comment]
@IamNotHuman please fix the failure above. thank you.

**Comment by groberts314** [comment]
@raoufr @IamNotHuman It's some kind of problem with `npm install` on the UI docs project :-(
I don't think it's an issue with this code change; it's some kind of problem with a dependency.
Someone will have to dig in and see what we can do to get it fixed.

https://dev.azure.com/saddlebackchurch/Church%20Management/_build/results?buildId=73004&view=logs&j=92a4755a-6173-591e-5c07-8dbe1165e937&t=e6427d9b-c390-5bdc-88e1-5cec3e52f9d8&l=11989

File: src/inputs/input/input.jsx [hunk 1]
```diff
@@ -14,6 +14,10 @@ import Icon from '../../dataDisplay/icon';
 import KeyCode from '../../global/keyCode';
 
 const propTypes = {
+    /**
+     * Indicates type of file input accepts.
+     */
+    accept: PropTypes.string,
```
Comment (War777) [inline src/inputs/input/input.jsx:20]: Where are we using this prop?

**Review by War777 — APPROVED** [review]

File: src/inputs/input/input.jsx (hunk 1 above)
Comment (IamNotHuman) [reply src/inputs/input/input.jsx]: removed

File: docs/package-lock.json [hunk 2]
```diff
@@ -3813,7 +3813,7 @@
         "jsontoxml": "^1.0.1",
         "lodash.defaultsdeep": "^4.6.1",
         "require-directory": "^2.1.1",
-        "sharp": "^0.23.3",
+        "sharp": "^0.23.2",
```
[the remark this answers is not in the document]
Comment (IamNotHuman) [reply docs/package-lock.json:3816]: you right missed that

**Comment by IamNotHuman** [comment]
@groberts314 or maybe this could work https://github.com/lovell/sharp/issues/2128#issuecomment-600232917

**Comment by groberts314** [comment]
@IamNotHuman No joy with the downgrade.  https://dev.azure.com/saddlebackchurch/Church%20Management/_build/results?buildId=73282&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=295ef4c3-3a46-5f11-020b-b8098f9465c0
Same error.

Maybe try an upgrade?

**Comment by groberts314** [comment]
> @groberts314 or maybe this could work [lovell/sharp#2128 (comment)](https://github.com/lovell/sharp/issues/2128#issuecomment-600232917)

Checking...
It seems weird to tell to it `--platform=linux` though.  We are building it on Windows.

**Comment by groberts314** [comment]
We have a winner!  Nice work @IamNotHuman.
:shipit:

**Pull request merged**